src/Libraries/CoreNodeModelsWpf/Charts/HeatSeriesNodeModel.cs

```diff
@@ -216,7 +216,7 @@ private void DataBridgeCallback(object data)
                 if (InPorts[3].IsConnected) return;
 
                 // In case colors are not provided, we supply some from the default library of colors
-                Info(Dynamo.Wpf.Properties.CoreNodeModelWpfResources.ProvideDefaultColorsWarningMessage);
+                Info(Dynamo.Wpf.Properties.CoreNodeModelWpfResources.ProvideDefaultColorsWarningMessage, true);
```

Comment:
again, I don't get it, will this be cleared when a user sets the values to non defaults?

Reply:
@mjkkirschner They are cleared once the user sets the values to nondefaults, the clearing logic is on a different place